test/messages.test.ts

```diff
@@ -93,7 +93,7 @@ describe("Messaging", () => {
 });
 
 describe("Message errors", () => {
-    const fakeAddress = "id1PNvUP4jHD74qo6usnWNoaFGFf33MXZi6c";
+    const fakeAddress = "did:e:fakeAddress:dids:0123456789012345678912";
```

Comment:
```suggestion
    const fakeAddress = "did:e:a-domain:dids:0000000000000000000000";
```
we should use this everywhere we use a valid but fake address.